superset-frontend/src/components/Alert/Alert.stories.tsx

```diff
@@ -87,6 +87,7 @@ InteractiveAlert.args = {
 InteractiveAlert.argTypes = {
   onClose: { action: 'onClose' },
   type: {
-    control: { type: 'select', options: types },
+    control: { type: 'select' },
+    options: types,
```

Comment:
This one doesn't seem to be working still. 
<img width="554" alt="image" src="https://github.com/apache/superset/assets/812905/8aebca6d-72d4-496f-97fe-4fb57c4dd459">

Reply:
This one modifies the type control, i think it should be working.